src/status_im/ui/screens/discover/events.cljs

```diff
@@ -1,148 +1,172 @@
+       [_ status]]
+    (if-let [hashtags (seq (handlers/get-hashtags status))]
+      (let [{:keys [name photo-path]} (get accounts current-account-id)
+            message    {:message-id random-id
+                        :from       current-public-key
+                        :payload    {:message-id random-id
+                                     :status     status
+                                     :hashtags   (vec hashtags)
+                                     :profile    {:name          name
+                                                  :profile-image photo-path}}}]
+
+        {::broadcast-status {:web3     web3
+                             :message  message
+                             :identities (handlers/identities contacts)}
+         :dispatch          [:status-received message]}))))
+
+(handlers/register-handler-fx
+  :init-discoveries
+  (fn [_ _]
+    {::delete-old-discoveries nil
+     :dispatch                [:request-discoveries]}))
+
+(handlers/register-handler-fx
+  :request-discoveries
+  [(re-frame/inject-cofx :random-id)]
+  (fn [{{:keys [current-public-key web3]
+         :contacts/keys [contacts]} :db
+        random-id :random-id} _]
+    ;; this event calls itself at regular intervals
+    ;; TODO: this was previously using setInterval explicitly, with
+    ;; dispatch-later it is using it implicitly. setInterval is
+    ;; problematic for such long period of time and will cause a warning
+    ;; for Android in latest versions of react-native
+    {::request-discoveries {:current-public-key current-public-key
+                            :web3               web3
+                            :identities         (handlers/identities contacts)
+                            :message-id         random-id}
+     :dispatch-later       [{:ms       (* request-discoveries-interval-s 1000)
+                             :dispatch [:request-discoveries]}]}))
+
+(handlers/register-handler-fx
+  :discoveries-send-portions
+  (fn [{:keys [db]} [_ to]]
+    (send-portions-when-contact-exists db to)))
+
+(handlers/register-handler-fx
+  :discoveries-request-received
+  (fn [{:keys [db]} [_ {:keys [from]}]]
+    (send-portions-when-contact-exists db from)))
+
+(handlers/register-handler-fx
+  :discoveries-response-received
+  [(re-frame/inject-cofx :now)]
+  (fn [{{:keys [discoveries]
+         :contacts/keys [contacts] :as db} :db
+        now :now}
+       [_ {:keys [payload from]}]]
+    (when (get contacts from)
+      (when-let [discovers (some->> (:data payload)
+                                    (filter #(new-discover? discoveries %))
+                                    (map #(assoc % :created-at now)))]
+        {:db (reduce add-discover db discovers)
+         ::add-discovers discovers}))))
+
+(handlers/register-handler-fx
+  :status-received
+  [(re-frame/inject-cofx :now)]
+  (fn [{{:keys [discoveries] :as db} :db
+        now :now}
+       [_ {{:keys [message-id status hashtags profile] :as payload} :payload
```

Comment:
align? I think

Reply:
I don't think we do that for destructuring